GEOS_ChemEnvGridComp.F90

```diff
@@ -826,57 +821,23 @@ subroutine Run2 ( GC, IMPORT, EXPORT, CLOCK, RC )
        end do
     end if
 
-! Import precip from MOIST or read observed precip from a file. 
+! Import total precip from SURFACE (either model or observed precip)
 ! Export total, convective and non-convective precipitation.
 !--------------------------------------------------------------
-    call MAPL_GetPointer ( EXPORT, tprec,    'TPREC',     __RC__ )
-    call MAPL_GetPointer ( EXPORT, cn_prcp,  'CN_PRCP',   __RC__ )
+    call MAPL_GetPointer ( EXPORT,    tprec,    'TPREC',  __RC__ )
```

Comment:
does this need to be 'PRECTOT' ??? (i.e., string used in export spec)
sorry if this is completely off base.

Reply:
Never mind... I got confused with imports and exports, only just noticed line 832 below.